utils/dashboard.js

```diff
@@ -91,6 +91,9 @@ const dashboard = async (req, res) => {
     } else if (api === 'participantDataCorrection') {
         const { participantDataCorrection } = require('./sites');
         return await participantDataCorrection(req, res);
+    } else if (api === "dryRunNotificationSchema") {
+        const { dryRunNotificaionSchema } = require('./notifications');
```

Comment:
you already caught the typo in notifications.js, but it assigned the update in the export